### PR TITLE
Blog Post: Calle LTS

### PR DESCRIPTION
**Pull request by dergigi**
**Build preview:**

- [/blog/cashu-calle-receives-lts-grant](https://os-website-git-calle-lts-open-sats.vercel.app/blog/cashu-calle-receives-lts-grant)

**Social preview:**

- 34-calle

### CODE REVIEW

File: data/blog/cashu-calle-receives-lts-grant.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+This LTS grant will support Calle's efforts to further enhance the robustness
+and performance of the Cashu protocol. Some of his key initiatives will include
+the introduction of a fee structure to prevent denial of service attacks,
+implementing a proof of liabilities scheme for auditing mints, and developing a
+"[multinut payments](https://github.com/cashubtc/nuts/pull/103)" concept to
+improve the utility of smaller Ecash balances.
+
+> Our mission with Cashu is to provide a robust, private, and efficient monetary
+> system that empowers users to transact with complete privacy and confidence.
```
Comment (Arvin21M) [inline data/blog/cashu-calle-receives-lts-grant.mdx:36]: > Our mission with Cashu is to create a robust ecash protocol built for Bitcoin with a focus on simplicity, efficiency, and privacy. Anyone should be able to build open-source applications powered by ecash. And anyone should be able to run them.

**Comment by dergigi** [comment]
Ready for final review by @OpenSats/volunteers

**Comment by btcschellingpt** [comment]
There's one phrase which may need amendment:
"and developing a multinut payments concept to improve the utility of smaller Ecash balances"

I understand that concept has already been discussed and talked about publicly, so this is less about developing the concept, as it is about building the implementation. Perhaps the following:

"and building the multinut payments concept to improve the utility of smaller Ecash balances"

**Comment by btcschellingpt** [comment]
@Arvin21M and @Bayernatoor .. lovely piece of work!

**Review by Arvin21M — APPROVED** [review]

**Comment by Bayernatoor** [comment]
Re-read again, checked links. All looks good. @btcschellingpt  good suggestion!

**Pull request merged**